src/components/Auth/LoginForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Box, Button, TextField } from '@material-ui/core/';
+import axios from 'axios';
+
+const LoginForm = ({ toggleActiveForm }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleLogin = e => {
+    e.preventDefault();
+    const data = {
+      username,
+      password,
+    };
+    axios
+      .post('https://localhost:8000/auth/obtain_token/', data)
```

Comment:
As I mentioned above (question #2), I wanted to make this be `/auth/obtain_token` (more suitable for production) instead of `localhost:8000`, but the proxy I put into `package.json` isn't working. Am stumped by this. :/